app/views/certificates.html

```diff
@@ -9,8 +9,32 @@
             <h2><i class="glyphicon glyphicon-certificate"></i>&nbsp;&nbsp;Certificates</h2>
           </div>
         </div>
+
+
+        <!-- Server Alerts -->
+        <alert ng-repeat="alert in alerts.server" type="alert.type">{{alert.msg}}</alert>
+        <!-- Server Alerts -->
         
-        <div class="exportSectionDiv" ng-show="currentServerCert && trustedCerts && certValidity && !alerts.server">
+          
+        <!-- Restart the server message -->
+        <alert ng-if="serverRestartRequired" class="alert ng-isolate-scope alert-danger" style="font-weight: bold; text-align: center; font-size: 16px;">
+          <div style="margin-bottom: 10px;">When adding/updating a certificate, a server restart is required for it to be applied.</div>
```

Comment:
Perhaps add a sentence here that says: "Please ensure that no critical transactions are running through the OpenHIM while restarting as these may be lost."